CordovaLib/Classes/Public/CDVURLSchemeHandler.m

```diff
@@ -39,45 +41,70 @@ - (void)webView:(WKWebView *)webView startURLSchemeTask:(id <WKURLSchemeTask>)ur
     NSURL * url = urlSchemeTask.request.URL;
     NSString * stringToLoad = url.path;
     NSString * scheme = url.scheme;
-
-    if ([scheme isEqualToString:self.viewController.appScheme]) {
-        if ([stringToLoad hasPrefix:@"/_app_file_"]) {
-            startPath = [stringToLoad stringByReplacingOccurrencesOfString:@"/_app_file_" withString:@""];
-        } else {
-            if ([stringToLoad isEqualToString:@""] || [url.pathExtension isEqualToString:@""]) {
-                startPath = [startPath stringByAppendingPathComponent:self.viewController.startPage];
-            } else {
-                startPath = [startPath stringByAppendingPathComponent:stringToLoad];
+    
+    CDVViewController* vc = (CDVViewController*)self.viewController;
+
+    /*
+     * Give plugins the chance to handle the url
+     */
+    BOOL anyPluginsResponded = NO;
+    BOOL handledRequest = NO;
+
+    for (NSString* pluginName in vc.pluginObjects) {
+        CDVPlugin* plugin = [vc.pluginObjects objectForKey:pluginName];
+        SEL selector = NSSelectorFromString(@"handleSchemeURL:");
+        if ([plugin respondsToSelector:selector]) {
+            handledRequest = (((BOOL (*)(id, SEL, id <WKURLSchemeTask>))objc_msgSend)(plugin, selector, urlSchemeTask));
+            NSLog(@"Handled: %d", handledRequest);
```

Comment:
Hi! Mind not adding NSLogs to code that can run in production environments?

Reply:
This has been removed on a later commit

Reply:
Ah, thanks for that-- (sorry) I googled in to the wrong place to begin my review of your PR.  Happy to discuss my other comment (which still applies), if clarification is required.

Reply:
Yeah the other comment ist still valid. We would need to fix more places though.

Would be nice if you could provide a pull request for the existing places. I am not am ObjC expert.

Reply:
Will do.  I am in the process of obtaining permission from my employer.